tests/ops/integration_tests/saas/connector_runner.py

```diff
@@ -340,15 +347,16 @@ async def _base_erasure_request(
             self.db,
         )
 
-        if (
-            ActionType.access
-            in SaaSConfig(**self.connection_config.saas_config).supported_actions
-        ):
-            # verify we returned at least one row for each collection in the dataset
-            for collection in self.dataset["collections"]:
-                assert len(
-                    access_results[f"{fides_key}:{collection['name']}"]
-                ), f"No rows returned for collection '{collection['name']}'"
+        if not skip_access_results_check:
```

Comment:
Quick fix to avoid checking for access data. This is necessary since our test account for Responsys isn't going to return access data for every collection.